app/Http/Controllers/Api/InventoryController1.php

```diff
@@ -0,0 +1,51 @@
+<?php namespace App\Http\Controllers;
```

Comment:
The namespace for this class should be `App\Http\Controllers\Api`. The Laravel framework needs the path to the class definition file to correspond with the class's namespace.